models/public/mosaic/Mosaic.md

```diff
@@ -0,0 +1,102 @@
+# mosaic
```

Comment:
You shouldn't call it just `mosaic` - that only identifies the variant. The model itself needs to be identified somehow. The directory in the repository it's from is called `fast_neural_style`, so `fast-neural-style-mosaic` would make sense. Also, `-onnx` should be appended to signify that this is an ONNX-converted model (not the original).

Reply:
fixed

Reply:
Model renamed to fast-neural-style-mosaic-onnx